src/xcode/ENA/ENA/Source/Models/Metadata/Test Result Metadata/__tests__/TestResultMetadataTests.swift

```diff
@@ -92,15 +92,16 @@ class TestResultMetadataTests: XCTestCase {
 		let riskCalculationResult = mockRiskCalculationResult(risk: .low)
 		secureStore.riskCalculationResult = riskCalculationResult
 
-		if let registrationDate = Calendar.current.date(byAdding: .day, value: -4, to: Date()) {
-			Analytics.collect(.testResultMetadata(.registerNewTestMetadata(registrationDate, "")))
-		} else {
+		guard let registrationDate = Calendar.current.date(byAdding: .day, value: -4, to: Date()) else {
 			XCTFail("registration date is nil")
+			return
 		}
-
+		Analytics.collect(.testResultMetadata(.registerNewTestMetadata(registrationDate, "")))
 		Analytics.collect(.testResultMetadata(.updateTestResult(.positive, "")))
 		XCTAssertEqual(secureStore.testResultMetadata?.testResult, TestResult.positive, "incorrect testResult")
-		XCTAssertEqual(secureStore.testResultMetadata?.hoursSinceTestRegistration, (24 * 4), "incorrect hoursSinceTestRegistration")
+		
+		let hours = Calendar.current.dateComponents([.hour], from: registrationDate, to: Date()).hour
```

Comment:
A question in generel. Would it help to run all the tests on a UTC timebase? You could use a Calendar in UTC timezone instead of the current, which involves timezone and so on.

Reply:
mmm, i think it will, but I'm not sure how big are the changes we need to do since most of the app deals with time

Reply:
I only see a timezone problem in the scope of this test.
By calculating 4 days back with a calendar, this test brings the current timezone into play.

The whole problem is, that this test depends on a timezone which can change, but is comparing with a fixed number of hours.